src/hooks/index.tsx

```diff
@@ -1,5 +1,6 @@
 import { useCallback, useEffect, useState } from 'react';
 
+/** @dev @TODO check if is useful or not (why would you want both state and ls unless it affects renders) */
```

Comment:
is there a more common way to use LS?

This was just a quick way I saw to generate functions that allowed functions to persist state to LS.

I don't think we have any uses of this where it affects renders, at the moment

Reply:
doing a hook to access LS is fine, what might be overhead is having the values both on state and LS , if put to state its usually to generate reactivity